src/vic.h

```diff
@@ -77,6 +77,9 @@
 extern char vic_buffer[VIC_BUFFER_SIZE + 1];
 extern uint8_t vic_buffer_len;
 
+#define vic_printf(format, ...) do { char _vic_buff[VIC_BUFFER_SIZE];\
```

Comment:
@mrshu I was not sure if I should make new constant or just write there some number. As large array as this is used also in vic_print_err, so I have idea to make something like vic_buffer_in and vic_buffer_out for input and output.